src/index.html

```diff
@@ -1,13 +1,57 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="ru">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="/CSS/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <main class="main">
+      <div class="card" data-qa="card">
+        <img
+          src="/images/imac.jpeg"
+          alt="imac"
+          class="card__image"
+        >
+
+        <h2 class="card__title">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </h2>
+
+        <div class="card__code">
+          Код товара: 195434
+        </div>
+
+        <div class="card__reviews">
+          <div class="reviews">
```

Comment:
Если необходимо использовать блок внутри другого блока - он должен быть одновременно и элементом внешнего блока:
```suggestion
        <div class="card__reviews reviews">
```

Reply:
.card__reviews - у меня там только марджин.
Если я добавлю его к .reviews, то весь блок у меня отскочит вниз.
Почему блок внутри блока обязан дублировать его классы?
Тогда мне нужно всем блокам так же присвоить еще и .main?